### ISSUE TITLE
near to, close to, far from: ADJ or ADV?

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#200]
PTB guidelines give "We were near/RB to the station" and "We were far/RB from home". The data looks inconsistent even for copular predicates though:

- [EWT](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=6112133c47caf&clustering=X.upos)
- [GUM](http://match.grew.fr/?corpus=UD_English-GUM@dev&custom=611213b48daec&clustering=X.upos)

**Comment by amir-zeldes** [comment]
Thanks, fixed - looks like in GUM it was only the 6 cases of "close to", right?

**Comment by nschneid** [comment]
Yeah. I take it if it is a copular predicate it should be ADV; if it is an amod dependent ("Texas multi-millionnaires close to the Bush family") it should be ADJ.

What about locative arguments of verbs: "put your fish tank close to the sun" and "I live pretty far from the office"?

**Comment by amir-zeldes** [comment]
I could see an argument for those to be ADV. I mean, put close and live far can both be replaced with "there" (put there, live there) and interrogated with "where", which are prototypical adverbs. OntoNotes has 2x"live far" and 2x"live close", and all are RB. 

I couldn't find one with "put", though "put there" is obviously RB and well attested. There is no instance of "put" dominating a JJ in a locative sense though.

**Comment by nschneid** [comment]
I looked at the PTB policy again, and it is a blanket statement that "near(er)/close(r)" in collocation with "to" is an adverb. TBH I'm not sure I see the rationale here. "Close" and "near" both have -ly counterparts. Why stipulate that "to" turns them from adjectives into adverbs?

Currently "multi-millionnaires close to the Bush family" uses amod(multi-millionaires, close). If we're changing "close" to ADV should it be `advmod`?

**Comment by nschneid** [comment]
CGEL classifies "near", "close", and "far" as both adjectives and (more commonly) prepositions (pp. 609-610). The following examples are offered as failing the predicand test for adjectives:

- This place is a dead end, but **near(er) the city** there's plenty going on.
- **Closer to election day,** the audience is much larger.
- **Not very far from their house** the road deteriorates into a dirt track.

Of course, CGEL takes a more expansive view of prepositions than UD does. So I guess these can be considered ADVs, and spatiotemporal ADVs postmodifying nouns are allowed (they are just PPs in CGEL). Which is an exception to the default assumption that modifiers of nouns are not adverbs.

**Comment by amir-zeldes** [comment]
> Currently "multi-millionnaires close to the Bush family" uses amod(multi-millionaires, close). If we're changing "close" to ADV should it be advmod?

I think there is a sense difference there: as ADV, I would read this as "mult-millionnaires, (**being**) close to the Bush family ...". But if it's just an attribute then I think it's a normal extraposed amod (like virtually all amods that take a PP child in English, e.g. "similar to").

> CGEL classifies "near", "close", and "far" as both adjectives and (more commonly) prepositions

For "near" I can understand it, but "close" by itself (or "far") is not a preposition - only if you make it fixed as "close to" and "far from", in which case the second word is what provides the prepositional syntactic behavior. The use of the entire complex as a PP does suggest that it's adverbial, since functionally there is no real external difference between a PP and an ADVP ("in the house" is interchangeable with "there")

**Comment by nschneid** [comment]
> > Currently "multi-millionnaires close to the Bush family" uses amod(multi-millionaires, close). If we're changing "close" to ADV should it be advmod?
> 
> I think there is a sense difference there: as ADV, I would read this as "mult-millionnaires, (**being**) close to the Bush family ...". But if it's just an attribute then I think it's a normal extraposed amod (like virtually all amods that take a PP child in English, e.g. "similar to").

I don't see a sense difference. It means the multimillionaires and the Bush family are close, as in close friends. Most postmodifying amods can be paraphrased with a modifier clause (*people afraid to go to work -> people who are afraid to go to work; people, being afraid to go to work, ...*).

> > CGEL classifies "near", "close", and "far" as both adjectives and (more commonly) prepositions
> 
> For "near" I can understand it, but "close" by itself (or "far") is not a preposition - only if you make it fixed as "close to" and "far from", in which case the second word is what provides the prepositional syntactic behavior. The use of the entire complex as a PP does suggest that it's adverbial, since functionally there is no real external difference between a PP and an ADVP ("in the house" is interchangeable with "there")

Locative "there" is also called a preposition in CGEL. :) Obviously this is different from UD terminology. I am just noting that we need to allow for some ADV/advmod postmodifiers of nouns because of "near to", "close to", "far from", and other spatially-related function words ("our time there", "our time together", "the paragraph below").

**Comment by amir-zeldes** [comment]
> I don't see a sense difference.

I think I didn't explain myself right: I would only see it as ADV if it was like a "genetivus absolutus" advcl, as in:

  * multi-millionnaires, (being) close to the Bush family, are prone to praise anyone called Bush

But that is clearly not what we have here, I agree this was just amod.

> Locative "there" is also called a preposition in CGEL

0_0

> we need to allow for some ADV/advmod postmodifiers of nouns

Absolutely, I've always been in favor of this, and there are plenty in GUM (examples: "information online", "the types above", "miles below", "Noggins here"...)

**Comment by nschneid** [comment]
"Strong-arm" is a tricky example of a compound verb where neither of the constituents is natively a verb. A more proper treatment of the MWE structure might be to have "arm" as a NOUN modified by "strong" and then use `ExtPos=VERB` on "arm". But I can live with the current analysis that "arm" itself has been coerced into a VERB. Would prefer `compound` over the current `advmod`, though.

**Comment by nschneid** [comment]
> > I don't see a sense difference.
> 
> I think I didn't explain myself right: I would only see it as ADV if it was like a "genetivus absolutus" advcl, as in:
> 
>     * multi-millionnaires, (being) close to the Bush family, are prone to praise anyone called Bush
> 
> 
> But that is clearly not what we have here, I agree this was just amod.

So are you saying we *shouldn't* follow the PTB's blanket rule that "close" in "close to" is always ADV?

I'm sort of ambivalent TBH. But I want an easy-to-apply policy.

**Comment by amir-zeldes** [comment]
Yes, I think it's clearly amod and ADJ, and I don't think there are enough PTB cases to establish that that's really what they would have done/worry that we're creating some huge incompatibility. In the predicative cases I'm fine with ADV.

**Comment by nschneid** [comment]
OK, so let's say the policy is:

> *close to*, *far to*, and *near to* are ADV+ADP if predicative ("We are close to home") and ADJ+ADP if modifying a nominal ("Sources close to the president say...").

I will update EWT.

**Comment by arademaker** [comment]
Hum, it seems you are using ExtPos=VERB for the same purpose we are using MWEPOS=VERB in the misc field in the Portuguese corpora.. the POS of the whole expression. Can you confirm? Should we try to adopt the same annotation?

**Comment by nschneid** [comment]
Yeah I think UniversalDependencies/docs#777 was where we first decided to adopt ExtPos for English.
